client/rpc-core/src/types/transaction_request.rs

```diff
@@ -95,6 +98,50 @@ mod access_list_item_camelcase {
 	}
 }
 
+/// Serde support for AuthorizationListItem with camelCase field names
+mod authorization_list_item_camelcase {
+	use ethereum::{eip2930::MalleableTransactionSignature, AuthorizationListItem};
+	use ethereum_types::{Address, H256};
+	use serde::{Deserialize, Deserializer};
+
+	#[derive(Deserialize)]
+	struct AuthorizationListItemDef {
```

Comment:
```suggestion
        #[serde(rename_all = "camelCase")]
	struct AuthorizationListItemDef {
```

Please use rename_all, so serde can rename fields to camelCase automatically instead of renaming each one manually; this prevents typos.